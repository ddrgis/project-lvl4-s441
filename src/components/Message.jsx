import React from 'react';

const Message = ({ message }) => (
  <div className="mb-1">
    <span className="font-weight-bold">
      {message.userName}
      {': '}
    </span>
    {message.message}
  </div>
);
export default Message;
