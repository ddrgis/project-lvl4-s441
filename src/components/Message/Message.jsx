import React from 'react';

const Message = ({ message }) => (
  <div>
    {message.userName}
    {': '}
    {message.message}
  </div>
);
export default Message;
