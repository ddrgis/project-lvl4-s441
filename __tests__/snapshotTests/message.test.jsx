import React from 'react';
import { shallow } from 'enzyme';
import Message from '../../src/components/Message';

it('renders correctly', () => {
  const stubMessage = { userName: 'name', message: 'hello' };
  const wrapper = shallow(<Message message={stubMessage} />);
  expect(wrapper).toMatchSnapshot();
});
