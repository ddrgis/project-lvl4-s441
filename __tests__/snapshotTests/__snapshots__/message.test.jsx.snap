// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mb-1"
>
  <span
    className="font-weight-bold"
  >
    name
    : 
  </span>
  hello
</div>
`;
